udata/core/dataset/search.py

```diff
@@ -8,25 +8,31 @@
 from udata.i18n import lazy_gettext as _
 from udata.core.site.views import current_site
 from udata.models import (
-    Dataset, Organization, License, User, GeoZone,
+    Dataset, Organization, License, User, GeoZone, CERTIFIED
 )
 from udata.search import (
     ModelSearchAdapter, i18n_analyzer, metrics_mapping_for, register,
 )
 from udata.search.fields import (
     TermsFacet, ModelTermsFacet, RangeFacet, TemporalCoverageFacet,
-    BoolBooster, GaussDecay, BoolFacet
+    BoolBooster, GaussDecay, BoolFacet, ValueFactor
 )
 from udata.search.analysis import simple
 
-from udata.core.spatial.models import spatial_granularities
+from udata.core.spatial.models import (
+    admin_levels, spatial_granularities, ADMIN_LEVEL_MAX
+)
 
 # Metrics are require for dataset search
 from . import metrics  # noqa
 
 __all__ = ('DatasetSearch', )
 
 
+# Number of years from which scoring is not impacted anymore
```

Comment:
Proposition: `# After this number of years, scoring is kept constant instead of increasing.`